src/common/main/kotlin/me/owdding/skyblockpv/api/CachedApis.kt

```diff
@@ -52,3 +54,14 @@ object ProfileAPI : CachedApi<UUID, List<SkyBlockProfile>, UUID>() {
         emptyList()
     }
 }
+
+object PlayerAPI : CachedApi<UUID, HypixelPlayer, UUID>() {
```

Comment:
player data probably wont change often so maybe caching this for much longer than 5 min would be good